chain/chain/src/flat_storage_resharder.rs

```diff
@@ -822,10 +638,7 @@ impl FlatStorageResharder {
             num_batches_done += 1;
             metrics.set_head_height(flat_head.height);
 
-            if postpone {
```

Comment:
I was on the fence about this, but realized we anyway have the loop + the sleep here for the retry.